clis/tcli/src/bin/dtcli/examples/example_vpn_disable.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Maybe leet's move examples to the root of tcli for better visability

Reply:
And probably nameing it contrib would be better not to confuse with rust examples

Reply:
Done